### ISSUE TITLE
Exponential back off on autorenew

### ISSUE THREAD

**Issue by DrogoNevets** [NginxProxyManager/nginx-proxy-manager#3575]
**Is your feature request related to a problem? Please describe.**
LetsEncrypt multiple failures cause the user to hit the rate limiter on LetsEncrypt

**Describe the solution you'd like**
Either the ability to turn off auto renew for certificates or exponential backoff after failures

**Describe alternatives you've considered**
Looked to see if i can turn auto renew off easily, but I cant see it

**Additional context**
I keep getting failures on auto renew, i suspect becasue of the ForceSSL bug mentioned elsewhere - but it appears to run every hour without fail. It would be great if I could control if it run at all, and if the auto runs used an exponential back off so as not to litter the logs etc

**Comment by tr1p0p** [comment]
Yeah, because of this bug cannot just... Config my domains with nginx proxy manager and have to wait to just have a working website. Kind of annoying
